### DISCUSSION TITLE
stuck during `npm i`

### DISCUSSION THREAD

**Discussion by z50913** [ddiu8081/chatgpt-demo#122] [Q&A] [answered]
### Describe the bug

C:/Desktop chatgpt-demo-main>npm i
C:\Uers\tom\Desktop\chatgpt-demo-main>npm i
idealTree:@babel/generator: timing idealTree:node_modules/@ba

卡着，重新运行了几次都卡在不同点位置，新装点node18版本点

### Reproduction

npm

### System Info

```Shell
win10 chrome
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ddiu8081/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/ddiu8081/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ddiu8081** [comment, marked as answer]
国内网络问题，可自行配置代理
